BaseFilter/sections/specific.txt

```diff
@@ -2019,6 +2019,7 @@ oploverz.asia##div[id^="oplov-"]
 oploverz.asia###sidebar > div.widget_text
 chillx.top###theotherads
 ||www.techpowerup.com/reviyuu/
+techpowerup.com##.s--item[br-cls]
```

Comment:
Looks like they start circumvent the rules. It also should be reported in https://github.com/AdguardTeam/AdGuardExtra.

Reply:
I see, thank you for post issue for AdGuardExtra.

Reply:
Should I close this pull request if adguard extra to handle?

Reply:
Yep, I'll close this due to https://github.com/AdguardTeam/AdGuardExtra/issues/401.

Reply:
ok,thank you.